applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -398,6 +415,48 @@ static struct bt_audio_stream_ops stream_ops = {
 	.recv = stream_recv_cb,
 };
 
+static void work_stream_start(struct k_work *work)
+{
+	int ret;
+	struct worker_data work_data;
+
+	ret = k_msgq_get(&kwork_msgq, &work_data, K_NO_WAIT);
+	if (ret) {
+		LOG_ERR("Cannot get info for start stream");
+	}
+
+	if (work_data.dir == BT_AUDIO_DIR_SINK) {
+		ret = bt_audio_stream_start(&headsets[work_data.channel_index].sink_stream);
+	} else if (work_data.dir == BT_AUDIO_DIR_SOURCE) {
+		ret = bt_audio_stream_start(&headsets[work_data.channel_index].source_stream);
+	}
```

Comment:
```suggestion
	} else {
	    LOG_ERR("Trying to use unknown direction");
	    bt_conn_disconnect(headsets[work_data.channel_index].headset_conn, BT_HCI_ERR_REMOTE_USER_TERM_CONN);
	    return;
	}
```

Reply:
Still don't see the else here, should be added to catch bad data

Reply:
This should be added